README.md

```diff
@@ -1 +1,256 @@
+- [PortFolio](https://santosh-konappanavar.github.io/PortFolio/)
+
+- Click here to see [loom video](https://www.loom.com/share/3174f93cb3414ca390828e69afc0a749)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Login to your git hub account.
+
+If you do not have create a git account using the link below.
+
+<a href ="wwww.github.com"> Create a Github Account</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  <li> cd my-folder
+  <li> git clone git@github.com:myaccount/my-project.git
+
+
+### Install
+
+Install this project with:
+
+<li> Open cloned folder on VSCODE and run the following codes on the terminal window
+<li> Run npm init -y 
+<li> Run npm install --save-dev hint@7.x
+<li> Run npx hint .
+<li> Run npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+<li> Run npx stylelint "**/*.{css,scss}"
+<li> Run npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+<li> Run npx eslint .
+
+### Usage
+
+#### To run the project, execute the following command:
+To run the project, open the following link:
+
+  - git clone
+  - npm install or npm init -y
+
+
+### Run tests
+
+This website has been completed and deployed
+
+### Deployment
+
+- [PortFolio](https://santosh-konappanavar.github.io/PortFolio/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+
+
+👤 **Author**
+
+- GitHub: [@Santosh-Konappanavar](https://github.com/Santosh-Konappanavar/Portfolio-mobile-setup)
+- Twitter: [@SantoshKonapp1](https://twitter.com/SantoshKonappa1)
+- LinkedIn: [@Santosh Konappanavar](https://www.linkedin.com/in/santosh-konappanavar/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-  More Project sections would be added.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+  
+  
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> I recently started my micronaut career. I would be glad to have any resource that would easy my journey in the program
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank Microverse Team for this opportunity. Its my solo project and I had  completed this project within deadline.
+
```

Comment:
- [ ] In this section you should give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).



![image](https://user-images.githubusercontent.com/75126481/219554289-8cbdc90d-c595-4ab0-87eb-5e0f02cd2316.png)